src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs

```diff
@@ -378,7 +378,7 @@ private void UpdateHumanAgentStats(Agent agent, AgentDrivenProperties props)
                 else if (equippedItem.WeaponClass is WeaponClass.Javelin or WeaponClass.ThrowingAxe or WeaponClass.ThrowingKnife or WeaponClass.Stone)
                 {
                     float unsteadyAccuracyPenaltyScaler = MBMath.ClampFloat((equippedItem.ThrustSpeed - 89.0f) / 13.0f, 0.0f, 1f);
-                    props.WeaponMaxUnsteadyAccuracyPenalty = props.WeaponInaccuracy;
+                    props.WeaponMaxUnsteadyAccuracyPenalty = 0f; // props.WeaponInaccuracy; = old value,
                     props.WeaponMaxMovementAccuracyPenalty = props.WeaponInaccuracy * 1.3f;

```

Comment:
Do not put comment that are not relevant to current status. These comments will make no sense in 3 years or in 5 years.
The git history is there for looking at past implementation , comments are not meant for that